source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseStakePoolDialog.tsx

```diff
@@ -118,7 +118,7 @@ const DelegationStepsChooseStakePoolDialog = observer((props: Props) => {
     activeStakePoolId = lastDelegatedStakePoolId;
   }
 
-  const selectedPoolTicker = selectedPool?.ticker;
+  const selectedPoolTicker = selectedPool?.ticker.toUpperCase();
```

Comment:
should we move this behavior to CSS as the other places are handling it outside JS?

Reply:
Ah but for this special case there is a reason: it only work over JS because these ticker values are actually passed to the i18n framework and rendered inside a paragraph.